source/documentation/appendix-docs/botor.md

```diff
@@ -94,13 +117,17 @@ See `?botor::s3_read` for additional options.
 
 `openxlsx::read.xlsx` will only open
 files with the `.xlsx` suffix but `s3_read` uses a temporary file without
-a suffix. To get around this either use `readxl::read_excel` or run the following for `openxlsx`
+a suffix. To get around this either use `readxl::read_excel` or run the 
+following for `openxlsx`
 
 ```r
-temp_loc <- tempfile(fileext=".xlsx")
-s3_download_file("s3://alpha-mybucket/my_data.xlsx", temp_loc)
-wb <- openxlsx::read.xlsx(temp_loc)
-unlink(temp_loc)
+s3_read_xlsx <- function(s3_path, ...) {
+    temp_loc <- tempfile(fileext=".xlsx")
+    s3_download_file(, temp_loc)
+    wb <- openxlsx::read.xlsx(s3_path, temp_loc, ...)
+    unlink(temp_loc)
+    wb
```

Comment:
```suggestion
    return(wb)
 }
```

Reply:
I wasn't sure what the R convention for this is, I've been playing with Rust recently which frowns on explicit returns in the last line of a function - 9653d41

Reply:
It was more the function wasn't closed off, rather than the return statement. 

Either works (return vs no return) and it depends who you ask as to what convention should be used.

Reply:
... and that's me writing too much Python - 13d92cb